src/components/LoanCollections/KivaClassicLoanCarousel.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+	<div class="tw-pt-4">
+		<transition name="kvfade">
+			<div
+				v-if="isLoading"
+				class="spinner"
+			>
+				<kv-loading-spinner />
+			</div>
+		</transition>
+
+		<kv-carousel
+			v-if="augmentedLoanIds.length > 0 && isVisible"
+			class="tw-w-full tw-overflow-visible md:tw-overflow-hidden"
+			:autoplay="false"
```

Comment:
This prop is no longer available on the new KivaClassic Carousel - there is no more autoplay